dspace/config/dspace.cfg

```diff
@@ -367,7 +367,7 @@ handle.dir = ${dspace.dir}/handle-server
 # Authorize collection administrators to manage item settings (for items owned by the collection)
 #core.authorization.collection-admin.item.delete = true
 #core.authorization.collection-admin.item.withdraw = true
-#core.authorization.collection-admin.item.reinstatiate = true
+#core.authorization.collection-admin.item.reinstantiate = true
```

Comment:
This configuration key core.authorization.collection-admin.item.reinstantiate is referenced in AuthorizeConfiguration::canCollectionAdminPerformItemReinstatiate - some DS instances could have been copied the key into local.cfg. If the key name is changed (corrected), a note needs to be added to the release notes that indicate this renaming.